doc/user/content/sql/create-materialized-view.md

```diff
@@ -7,6 +7,15 @@ menu:
     parent: 'commands'
 ---
 
+{{< note >}}
+
+As an alternative to materialized views, [indexes on
+views](/concepts/views/#views) **maintain and incrementally update view results
+in memory** for the cluster where you create the index.  For more information,
+see [Views](/concepts/views/#views) and [`CREATE VIEW`](/sql/create-view).
+
+{{</ note >}}
```

Comment:
```suggestion
{{< note >}}
As an alternative to materialized views, [indexes on views](/concepts/views/#views)
**maintain and incrementally update view results in memory** for the cluster
where you create the index.  For more information, see [Views](/concepts/views/#views)
and [`CREATE VIEW`](/sql/create-view).
{{</ note >}}
```

Reply:
This might be more useful with some more actionable context. "Depending on your use case, you might prefer to create an indexed view. For more information on best practices, see <page>."